src/RolloutScorer/RolloutScorerAzureFunction/Configs.cs

```diff
@@ -0,0 +1,67 @@
+using RolloutScorer;
+using System.Collections.Generic;
+
+namespace RolloutScorerAzureFunction
+{
+    public static class Configs
+    {
+        public static readonly Config DefaultConfig = new Config
+        {
+            RepoConfigs = new List<RepoConfig>()
+            {
+                new RepoConfig
+                {
+                    Repo = "dotnet-helix-service",
+                    DefinitionId = "620",
+                    AzdoInstance = "dnceng",
+                    GithubIssueLabel = "Rollout Helix",
+                    ExpectedTime = 30,
+                    ExcludeStages = new List<string>() { "Post_Deployment_Tests" },
```

Comment:
This should be updated before merging; this is a remnant from when arcade-services used releases. What are the current stages we need to exclude?

Reply:
There are quite a few now.  I'm not sure what they are all called, or what we want to include when. :-(

Reply:
We already have the table that includes all the deployment times... can we just use that and not do this part?

Reply:
I'll look into switching to it but I think it's probably for the best.

Reply:
@ChadNedzlek how is the deployment time in that table calculated? Does it include post-deployment steps and/or approval time?

Reply:
The table it triggered by 2 tasks in the build, so it only works for arcade services. Right now it's only around the "deployment" part of the build, so I'm not sure if it will work for this purpose (it was mostly designed to trigger a "if you see a strange alert, it was during a deployment, so that's probably relevant").  We could... augment it if that's more useful?

Reply:
The "two tasks in the build" method is ideal imo because it lets us very strictly control what build time is. We'll adjust this once that's added (filed as https://github.com/dotnet/core-eng/issues/8565), but for now I'm going to resolve this since I can't do it across all repos we're scoring.